packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -164,6 +164,13 @@ export type CallCompositeOptions = {
    * if this is not supplied, the composite will not show a unsupported browser page.
    */
   onEnvironmentInfoTroubleshootingClick?: () => void;
+  /* @conditional-compile-remove(pinned-participants) */
+  /**
+   * If set to false, remote video tiles in the VideoGallery will not have menu options
+   *
+   * default: true
+   */
+  showRemoteVideoTileMenu?: boolean;
```

Comment:
from review board. We always try default to false (as then it is the same as not supplying the property), so this should be `hideRemoteVideoTileMenu` with `@default false`

Reply:
This should apply to both I believe.